store.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <title>Karma</title>
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:400,500,300"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+  </head>
+
+  <body>
+    <header class="header">
+      <img src="img/karma-logo.svg" alt="Karma Logo" class="nav-logo" />
+      <nav>
+        <ul class="nav-list">
+          <li class="nav-item-meet-karma">
+            <a class="nav-link-meet-karma" href="index.html">Meet Karma</a>
+          </li>
+          <li>
+            <a href="#">How it Works</a>
+          </li>
+          <li>
+            <a href="store.html">Store</a>
+          </li>
+          <li>
+            <a href="#">Blog</a>
+          </li>
+          <li>
+            <a href="#">Help</a>
+          </li>
+          <li>
+            <a href="">Login</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <section id="store-section">
+        <form class="form-container">
+          <p>Order your Karma wifi device today!</p>
+
+          <fieldset class="name">
+            <div class="name-group">
+              <label for="firstname">First name *</label>
```

Comment:
There is a way to add that required symbol (*) using CSS style, so you don't need to add it manually all the time:

```html
<label class="label-padding required" for="name">First name</label>
```

CSS:

```css
.required:after {
  content:" *";
}
```